### PR TITLE
init express api + mws + dotenv

### PR DESCRIPTION
**Pull request by geni94**

### CODE REVIEW

**Review by lincolnaleixo — COMMENTED** [review]
Like we discussed, I think you're using the legacy amazon-mws library and we don't want that.

File: package.json [hunk 1]
```diff
@@ -19,13 +19,14 @@
   },
   "homepage": "https://github.com/lincolnaleixo/Seller-Partner-API#readme",
   "dependencies": {
-    "amazon-mws": "^0.0.27",
     "amazon-sp-api": "^0.5.1",
```
Comment (lincolnaleixo) [inline package.json:22]: It seems there're an already wrapper so we won't need to do the wrapper by ourselves?

**Review by lincolnaleixo — CHANGES_REQUESTED** [review]
Please confirm if the amazon-sp-api wrapper is enough . This will make our project faster to implement. I'm ok with it if you checked their code and verify it's good. If so we may remove this from our tasks and remove the other repository

File: package.json (hunk 1 above)
Comment (geni94) [reply package.json:22]: @lincolnaleixo the idea was to not simply create a wrapper for Amazon SP, but also an API that will work with the front end and deliver the desired functionalities. I would not recommend exposing or using this directly on the front end application, and this way we have a lot more control on what is going on. Additionally, there should be a token exchange (for the authorization), and it would be better to handle this appropriately in multiple levels/tiers, not just on the front end.
So, in short, I don't think there is any repo to be removed.

**Comment by geni94** [comment]
commit `1b36d5ebe5e48ca7fd9e7fde94156b6b65f064cb` adds the Dockerfile and a small edit for the request headers to handle CORS.

**Comment by lincolnaleixo** [comment]
As we discussed on the chat we are not going to use this as monorepo
